doc/nrf/known_issues.rst

```diff
@@ -212,6 +286,22 @@ Subsystems
 |BLE|
 =====
 
+.. rst-class:: v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+NCSDK-6845: Pairing failure with simultaneous pairing on multiple connections
+  When using LE Secure Connections pairing, the pairing fails with simultaneous pairing on multiple connections.
+  The failure reason is unspecified.
+
+  **Workaround:** Retry the pairing on the connections that failed one by one after the pairing procedure has finished.
+
+.. rst-class:: v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
```

Comment:
This issue has been fixed for 1.4.1

See bottom:
`* Fixed an issue where a pairing fail could lead to a GATT procedure failure.`